src/Schema/DataObject/Plugin/DBFieldArgs/DBDateArgs.php

```diff
@@ -48,7 +45,7 @@ protected function getResolver(): callable
      * @return DBField | string
      * @throws Exception
      */
-    public static function resolve($obj, array $args)
+    public static function resolve(mixed $obj, array $args): DBField|string
```

Comment:
Not supported in PHP7.4

```suggestion
    public static function resolve($obj, array $args)
```